src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const spiderWidth = 50;
+  const board = 400;
```

Comment:
it would be better not to hardcode these values to have the possibility to change them using CSS. 
```suggestion
  const spiderWidth = spider.offsetWidth;
  
  const board = document.querySelector('.wall');
  const boardWidth = board.offsetWidth;
  const boardHeight = board.offsetHeight;
```